src/librustc/mir/repr.rs

```diff
@@ -688,14 +688,23 @@ pub struct Statement<'tcx> {
 
 #[derive(Clone, Debug, RustcEncodable, RustcDecodable)]
 pub enum StatementKind<'tcx> {
+    /// Write the RHS Rvalue to the LHS Lvalue.
     Assign(Lvalue<'tcx>, Rvalue<'tcx>),
+
+    /// Start a live range for the storage of the local.
+    StorageLive(Lvalue<'tcx>),
```

Comment:
Hmm, should this be restricted to base lvalues or something like that? Meh, I guess it's ok to use `Lvalue<'tcx>` here.